SampleData/Snapshot-SampleData.ps1

```diff
@@ -0,0 +1,98 @@
+param(
+	[string]$eproRoot = $null,
+	[string]$version = "22.02.0-beta"
+)
+$ErrorActionPreference = "Stop";
+
+if (-not $eproRoot) {
+	$versionPart = $version.Replace(".", "_");
+	$eproRoot = "C:\Program Files\Epro_${versionPart}";
+}
+
+if (-not (Test-Path $eproRoot)) {
+	Write-Host "Path does not exist: '${eproRoot}'";
+	exit 2;
+}
+
+#
+# First, set up and verify our environment.
+#
+
+$thisDir = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent;
+$eproConfigPath = "${eproRoot}\Web\bin\Epro.config";
+$eproDataPath = "C:\EproDocs";
+$stashToolPath = Get-Item "${thisDir}\tools\Bluewire.Stash.Tool.exe";
+$snapshotToolPath = Get-Item "${eproRoot}\Web\bin\Bluewire.Epro.Snapshot.exe";
+$remoteStashRoot = "https://bluewirestashservice20210521130718.azurewebsites.net";
+$stashName = "Bluewire.Epro.ApplySampleData";
+
+if (-not (Test-Path "${eproConfigPath}")) {

```

Comment:
should not need string interpolation here

Reply:
Generally speaking with shells, it's a good idea to wrap absolutely everything which should be a string in some kind of string interpolation. While it's not _necessary_ with PowerShell like it is with POSIX shell, it's not something to avoid.